wear/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -25,6 +25,12 @@
     @Override
     public void onCreate() {
         super.onCreate();
+
+        //KS copied from app/Home
+        Home.context = getApplicationContext();
+        set_is_follower();
```

Comment:
Yes, that's right, wear/BgReading is used when wear watch is running in standalone-g5 mode.  Yes, I wanted to keep wear/BgReading as similar to app/BgReading as possible so I reproduced calls in wear/Home for follower.  I think it is safe to set follower to false, which is what I was originally doing:)